src/packet.rs

```diff
@@ -63,6 +63,19 @@ impl Default for Packet {
     }
 }
 
+pub trait Recycle {
+    // Recycle trait is an object that can re-initialize important parts
+    //  of itself, similar to Default, but not necessarily a full clear
+    //  also, we do it in-place.
+    fn reset(&mut self) -> ();
```

Comment:
Did that ` -> ()` get by Clippy?  Can you remove it?

Reply:
can remove it

Reply:
clippy keeps going away on my box(es) ...rust updates?

Reply:
looks like clippy doesn't care, though https://github.com/solana-labs/solana/blob/master/src/crdt.rs#L338